steampy/client.py

```diff
@@ -278,7 +278,7 @@ def make_offer_with_url(self, items_from_me: List[Asset], items_from_them: List[
         }
         headers = {'Referer': self.COMMUNITY_URL + urlparse.urlparse(trade_offer_url).path,
                    'Origin': self.COMMUNITY_URL}
-        response = self._session.post(url, data=params, headers=headers).json()
+        response = self._session.post(url, data=params, headers=headers).json() or {}
```

Comment:
Maybe we should add new Exception, something for checking if status was 4xx and the raise ClientErrorException, similar for 5xx - Server error Exception, this would be better, if you like add this exception and manualy check it here. I will add auto checking for bad status in response later